index.html

```diff
@@ -933,8 +916,8 @@ <h3>Cross-origin Resources</h3>
 <a data-cite=
 "PERFORMANCE-TIMELINE-2#performance-timeline">Performance
 Timeline</a>. If the <a>timing allow check</a> algorithm fails for
-a <a>cross-origin</a> resource, these attributes of its
-<a>PerformanceResourceTiming</a> object MUST be set to zero:
+a resource, these attributes of its <a>PerformanceResourceTiming</a>
```

Comment:
removed the "cross-origin" wording, as we want the check to run on same-origin resources as well